### PR TITLE
[DSD-2392] Added testrigs-cron helm chart.

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

File: charts/testrigs-cron/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            serviceAccountName: pod-deleter
+            restartPolicy: Never
+            containers:
+              - name: kubectl
+                image: bitnami/kubectl
+                command:
+                   - bash
+                   - -c
+                   - "kubectl rollout restart deployment {{ .Values.deployments.deploymentName_1 }} -n {{ .Values.namespaces.namespace_1 }} &&
+                      kubectl rollout restart deployment {{ .Values.deployments.deploymentName_2 }} -n {{ .Values.namespaces.namespace_2 }}"
```
Comment (syedsalman3753) [inline charts/testrigs-cron/templates/cronjob.yaml:23]: Please add the rollout status check 

```kubectl -n $NS rollout status deployment/deployment-name```

File: charts/testrigs-cron/templates/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Values.cronjob.name }}
+spec:
+  concurrencyPolicy: Replace
```
Comment (syedsalman3753) [inline charts/testrigs-cron/templates/cronjob.yaml:6]: reason why concurrencyPolicy is set to replace?

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
Please rename the chart with 'restart-deployment-cronjob`

**Review by Rakshitha650 — APPROVED** [review]

File: charts/dslorchestrator/values.yaml [hunk 3]
```diff
@@ -442,8 +443,7 @@ dslorchestrator:
     dslorchestrator:
       ENABLE_INSECURE: "false"
       ENDPOINT: "https://api-internal.sandbox.xyz.net"
-      # TESTLEVEL: sanity or full
-      TESTLEVEL: "full"
+      TESTLEVEL: "full,sanity"
```
Comment (ckm007) [inline charts/dslorchestrator/values.yaml:446]: remove this as it is not getting used in templates.

File: charts/testrigs-cron/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: testrigs-cron
+description: A Helm chart to deploy dslorchestrator to test working of MOSIP modules
```
Comment (ckm007) [inline charts/testrigs-cron/Chart.yaml:3]: update the description

**Pull request closed without merging**